HKMP/Game/Client/PlayerManager.cs

```diff
@@ -52,30 +57,60 @@ internal class PlayerManager {
 
             _playerData = playerData;
 
-            // Create the player prefab, used to instantiate player objects
-            _playerPrefab = new GameObject(
-                "PlayerPrefab",
-                typeof(BoxCollider2D),
-                typeof(DamageHero),
-                typeof(EnemyHitEffectsUninfected),
-                typeof(MeshFilter),
-                typeof(MeshRenderer),
-                typeof(NonBouncer),
-                typeof(SpriteFlash),
-                typeof(tk2dSprite),
-                typeof(tk2dSpriteAnimator),
-                typeof(CoroutineCancelComponent)
-            ) {
-                layer = 9
-            };
-
-            // Add some extra gameObjects related to animation effects
-            new GameObject("Attacks") {layer = 9}.transform.SetParent(_playerPrefab.transform);
-            new GameObject("Effects") {layer = 9}.transform.SetParent(_playerPrefab.transform);
-            new GameObject("Spells") {layer = 9}.transform.SetParent(_playerPrefab.transform);
-
-            _playerPrefab.SetActive(false);
-            Object.DontDestroyOnLoad(_playerPrefab);
+            for (ushort playerId = 0; playerId <= MaxPlayers; playerId++) {
+                // Create a player container
+                var playerContainer = new GameObject($"Player Container {playerId}");
+
+                playerContainer.AddComponent<PositionInterpolation>();
+
+                // Create the player prefab, used to instantiate player objects
+                var playerPrefab = new GameObject(
+                    "PlayerPrefab",
+                    typeof(BoxCollider2D),
+                    typeof(DamageHero),
+                    typeof(EnemyHitEffectsUninfected),
+                    typeof(MeshFilter),
+                    typeof(MeshRenderer),
+                    typeof(NonBouncer),
+                    typeof(SpriteFlash),
+                    typeof(tk2dSprite),
+                    typeof(tk2dSpriteAnimator),
+                    typeof(CoroutineCancelComponent)
+                )
+                {
```

Comment:
Code-style inconsistency.
```suggestion
                ) {
```